avm/res/network/firewall-policy/main.bicep

```diff
@@ -41,13 +41,13 @@ param bypassTrafficSettings array?
 @description('Optional. List of specific signatures states.')
 param signatureOverrides array?
 
-@description('Optional. The configuring of intrusion detection.')
+@description('Optional. The configuring of intrusion detection. Default is `Deny`.')
 @allowed([
   'Alert'
   'Deny'
   'Off'
 ])
-param mode string = 'Off'
```

Comment:
Hey @ChrisSidebotham, as this is a breaking change, I'd recommend updating the `version.json`.

Reply:
Done